tests/PHPStan/Rules/Properties/data/missing-readonly-property-assign-phpdoc.php

```diff
@@ -109,3 +109,16 @@ public function __construct(int $foo)
 	}
 
 }
+
+class TestCase
```

Comment:
ups, to be correct this should go in the previous commit..

Reply:
nope, it should not be here at all, as it exists already above. this was a stupid copy/paste kind of mistake